code/modules/client/preference_setup/loadout/loadout_xeno.dm

```diff
@@ -99,12 +99,20 @@
 	sort_category = "Xenowear"
 
 /datum/gear/mask/vaurca
-	display_name = "mandible garment (Vaurca)"
+	display_name = "mask selection (Vaurca)"
 	path = /obj/item/clothing/mask/breath/vaurca
 	cost = 1
 	whitelisted = list("Vaurca Worker", "Vaurca Warrior",)
 	sort_category = "Xenowear"
 
+/datum/gear/mask/vaurca/New()
+	..()
+	var/masks = list()
+	masks["mandible garment"] = /obj/item/clothing/mask/breath/vaurca
+	masks["human expression mask"] = /obj/item/clothing/mask/breath/vaurca/expression
+	masks["vaurca expression mask"] = /obj/item/clothing/mask/breath/vaurca/expression/skrell
```

Comment:
Don't you mean masks["skrell expression mask"]? An expression mask of an uenxpressive alien seems expressly unexpressing.